Formula/m/mender-artifact.rb

```diff
@@ -0,0 +1,41 @@
+class MenderArtifact < Formula
+  desc "CLI tool for managing Mender artifact files"
+  homepage "https://mender.io"
+  url "https://github.com/mendersoftware/mender-artifact.git", tag: "4.1.0", revision: "af3853ecb7aae71c535fb564b4145b98ff7265f6"
```

Comment:
using github tarball should be fine.
```suggestion
  url "https://github.com/mendersoftware/mender-artifact/archive/refs/tags/4.1.0.tar.gz"
  sha256 "d82cd2f802033d53f2e947ed8d9d6cdd7a036fadbd92a2696b72122bd2070039"
```